docs/pages/docs/query/sql-client.mdx

```diff
@@ -150,40 +136,208 @@ function App() {
 
-```tsx
+const result = await client.db
+  .select()
+  .from(schema.account)
+  .orderBy(desc(schema.account.balance));
+```
+
+```ts [index.ts (@ponder/react)]
+import { desc } from "@ponder/client";
 import { usePonderQuery } from "@ponder/react";
-import { schema } from "../lib/ponder";
- 
-export function Deposits() {
-  const { data, isError, isPending } = usePonderQuery({
-    queryFn: (db) =>
-      db.select()
-        .from(schema.depositEvent)
-        .orderBy(schema.depositEvent.timestamp)
-        .limit(10),
-  });
- 
-  if (isPending) return <div>Loading deposits</div>;
-  if (isError) return <div>Error fetching deposits</div>;
-  return <div>Deposits: {data}</div>;
-}
+import * as schema from "../../ponder/ponder.schema";
+
+const query = usePonderQuery({
+  queryFn: (db) =>
+    db.select().from(schema.account).orderBy(desc(schema.account.balance)),
+});
 ```
 
-::::
+:::
+
+:::tip
+The `@ponder/client` package re-exports all Drizzle utility functions (like `desc` and `eq`). You shouldn't need to install `drizzle-orm` separately.
+:::
+
+### Live queries
+
+Live queries are automatically updated when the underlying data changes. Ponder uses Server-Sent Events to stream updates to the client every time a new block is processed.
+
+:::code-group
+
+```ts [index.ts (@ponder/client)] 
+import { desc } from "@ponder/client";
+import * as schema from "../../ponder/ponder.schema";
+
+await client.live(
+  (db) => 
+    db.select().from(schema.account).orderBy(desc(schema.account.balance)),
+  (result) => {
+    // ... handle result
+  },
+  (error) => {
+    // ... handle error
+  },
+);
+```
+
+```ts [index.ts (@ponder/react)]
+import { desc } from "@ponder/client";
+import { usePonderQuery } from "@ponder/react";
+import * as schema from "../../ponder/ponder.schema";
+
+// `usePonderQuery` always uses live queries!
+const query = usePonderQuery({
+  queryFn: (db) =>
+    db.select().from(schema.account).orderBy(desc(schema.account.balance)),
+});
+```
+
+:::
+
+### Untyped queries
+
+It's also possible to query the Ponder database without relying on importing `ponder.schema.ts` using the [`sql`](https://orm.drizzle.team/docs/sql) operator.
+
+:::code-group
+
+```ts [index.ts (@ponder/client)] 
+import { sql } from "@ponder/client";
+
+const result = await client.db.execute(sql`SELECT * FROM account limit 10;`);
+```
+
+```ts [index.ts (@ponder/react)]
+import { sql } from "@ponder/client";
+import { usePonderQuery } from "@ponder/react";
+
+const query = usePonderQuery({
+  queryFn: (db) => db.execute(sql`SELECT * FROM account limit 10;`),
+});
+```
+
+:::
+
+### Pagination
+
+The SQL client supports both limit/offset and cursor-based pagination patterns to handle large result sets efficiently.
+
+:::code-group
+
+```ts [index.ts (@ponder/client)] 
+import { desc } from "@ponder/client";
+import * as schema from "../../ponder/ponder.schema";
+
+const count = await client.db.$count(schema.account);
```

Comment:
What does this line demonstrate wrt pagination?

Reply:
Count is often used in pagination to know how many total pages there are